src/Hedgehog/Linq/Property.fs

```diff
@@ -71,91 +71,46 @@ type PropertyExtensions private () =
         let (Property property) = property
         Property.report property
 
```

Comment:
It might actually be nice to keep the `config` overloads. Thoughts?

Reply:
I think we should only expose a single way to configure properties; this new API should be the only API.

Reply:
I am not exactly sure what is being discussed here.  However, I am not overly concerned with the C# API.